x/delayedack/types/genesis.go

```diff
@@ -1,17 +1,12 @@
 package types
 
-import (
-	commontypes "github.com/dymensionxyz/dymension/v3/x/common/types"
-)
-
 // DefaultIndex is the default global index
 const DefaultIndex uint64 = 1
 
 // DefaultGenesis returns the default genesis state
 func DefaultGenesis() *GenesisState {
 	return &GenesisState{
-		Params:         DefaultParams(),
-		RollappPackets: []commontypes.RollappPacket{},
+		Params: DefaultParams(),
```

Comment:
shouldn't we remove rollapppackets from genesistate proto?

Reply:
prefer not to change proto files, as it requires store migration on upgrade.
we also **will support** hub forking in the future, once we'll research the required flow from RollApp prespective